backend/store.py

```
@@ -1510,12 +1749,258 @@ def clear_config(self):
+            Optional[Any]: Config value if it exists, None otherwise.
+        """
+        redis_key = self._posthog_config_key(project_id, key)
+        raw = self._get(redis_key)
+        if not raw:
+            return None
+        try:
+            return json.loads(raw)
+        except json.JSONDecodeError:
+            return None
+
         if keys:
```

Ruff (0.14.8)

Undefined name `keys`

(F821)